.circleci/config.yml

```diff
@@ -264,16 +245,18 @@ jobs:
       MESSAGE_PAYLOAD_FILE: "/tmp/message_payload"
     steps:
       - build_setup
-      - restore_cargo_package_cache
       - attach_workspace:
           at: /home/circleci/project
+      - restore_cargo_package_cache
```

Comment:
Got it, this is save since the cargo cache is placed in /usr/local/cargo, but this was an unnecessary change.  Puzzled on it for a bit.

Reply:
This restores the crate cache, not saving it. It is needed to speed up the build by avoiding fetching over network.

The two lines (`attach_workspace blah blah`) right above this are actually not needed anymore, as there is no more prebuilt binary to restore from a prev job.

Reply:
Fixed in #4016